services/gateway/apis/http-version2/swagger/definitions/meta.json

```diff
@@ -0,0 +1,74 @@
+{
+    "Pagination": {
+        "type": "object",
+        "required": [
+            "count",
+            "offset"
+        ],
+        "properties": {
+            "count": {
+                "type": "integer",
+                "example": 100,
+                "minimum": 0,
+                "description": "Number of items returned in the request."
+            },
+            "offset": {
+                "type": "integer",
+                "example": 25,
+                "minimum": 0,
+                "description": "Defines how many items are skipped."
+            },
+            "total": {
+                "type": "integer",
+                "example": 43749,
+                "minimum": 0,
+                "description": "Total number of items in the database."
+            }
+        }
+    },
+    "meta": {
+        "type": "object",
+        "required": [
+            "count",
+            "offset"
+        ],
+        "properties": {
+            "count": {
+                "type": "integer",
+                "example": 100,
+                "minimum": 0,
+                "description": "Number of items returned in the request."
+            },
+            "offset": {
+                "type": "integer",
+                "example": 25,
+                "minimum": 0,
+                "description": "Defines how many items are skipped."
+            },
+            "total": {
+                "type": "integer",
+                "example": 43749,
+                "minimum": 0,
+                "description": "Total number of items in the database."
+            },
+            "address": {
+                "type": "string",
+                "format": "address",
+                "example": "4935562234363081651L",
+                "description": "Lisk Address of the delegate the queried account voted for."
+            },
+            "publicKey": {
+                "type": "string",
+                "format": "publicKey",
+                "example": "968ba2fa993ea9dc27ed740da0daf49eddd740dbd7cb1cb4fc5db3a20baf341b",
+                "description": "Public key of the delegate the queried account voted for."
+            },
+            "username": {
+                "type": "string",
+                "format": "username",
+                "example": "liskhq",
+                "description": "Username of the delegate the queried account voted for."
+            }
```

Comment:
This is endpoint-specific, could you please attach it to the custom endpoint? The general pagination would be enough as a common definition.